code/_onclick/hud/screentip.dm

```diff
@@ -0,0 +1,19 @@
+/obj/screen/screentip
+	icon = null
+	icon_state = null
+	mouse_opacity = MOUSE_OPACITY_TRANSPARENT
+	screen_loc = "TOP,LEFT"
+	maptext_height = 480
+	maptext_width = 480
+	maptext = ""
+
+/obj/screen/screentip/Initialize(mapload, _hud)
+	. = ..()
+	hud = _hud
+	update_view()
+
+/obj/screen/screentip/proc/update_view(datum/source)
+	SIGNAL_HANDLER
```

Comment:
Why is `SIGNAL_HANDLER` here if this is never called from a signal?

Reply:
I forgot to remove it... thanks